e2e/client/playwright/page-object-models/authoring.ts

```diff
@@ -21,4 +21,18 @@ export class Authoring {
             .getByRole('button', {name: actionPath[actionPath.length - 1]})
             .click();
     }
+
+
+    async openMediaMetadataEditor() {
+        await this.page.locator(s('image-overlay')).hover();
+        await this.page.locator(s('edit-metadata')).click();
+    }
+
+    field(field: string) {
+        return this.page.locator(s('authoring', 'field--' + field)).getByRole('textbox');
+    }
+
+    async waitForAutosave() {
```

Comment:
I see this is not used in the end. Can we drop it? It's a hack anyway, in 90% of cases tests should work solely with the DOM. API mocking can be useful to test integration with 3rd party services, but other than that we should avoid interaction with HTTP requests.